### PR TITLE
Describe the "toggle" field config value

### PR DESCRIPTION
**Pull request by bytehead**
See https://github.com/contao/contao/blob/cd935664582cc3883d8d465fbdf6ecef29b3bce0/core-bundle/src/Resources/contao/drivers/DC_Table.php#L2743-L2746

### CODE REVIEW

**Review by fritzmg — COMMENTED** [review]
I think the `toggle` option is only present in Contao 5+, correct?

**Comment by bytehead** [comment]
Nope, also in 4.13

**Comment by bytehead** [comment]
https://github.com/contao/contao/blob/4.13/core-bundle/src/Resources/contao/drivers/DC_Table.php#L2743-L2746

File: docs/dev/reference/dca/fields.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ $GLOBALS['TL_DCA']['tl_example']['fields']['myfield'] = [
...
 | exclude              | true/false (`bool`)                             | If true the field will be excluded for non-admins. It can be enabled in the user group module (allowed excluded fields).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| toggle               | true/false (`bool`)                             | If true the field can be used to trigger the `toggle` action.
```
Comment (fritzmg) [inline docs/dev/reference/dca/fields.md]:
```suggestion
| toggle               | true/false (`bool`)                             | {{< version-tag "4.13" >}} If true the field can be used to trigger the `toggle` action.
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Comment by fritzmg** [comment]
Thank you @bytehead 

I also merged https://github.com/contao/docs/pull/1146

**Pull request merged**